backend/bin/deal-observer-backend.js

```diff
@@ -12,7 +12,7 @@ const { INFLUXDB_TOKEN } = process.env
 if (!INFLUXDB_TOKEN) {
   console.error('INFLUXDB_TOKEN not provided. Telemetry will not be recorded.')
 }
-const LOOP_INTERVAL = 10 * 1000
+const LOOP_INTERVAL = 100 * 1000
```

Comment:
what is the motivation for this change? Side note: There is an easier way to write this:

```suggestion
const LOOP_INTERVAL = 100_000
```

Reply:
The motivation was that during running the application we saw a lot of prints for how long the loop lasted. There were two options: 

1. Increase the loop duration
2. Only print the loop duration very X rounds

The latter would result in a loss of information which is why I chose the former. Inside the loop we retrieve all blocks up to the current head anyway, so it does not matter that much whether we ask for blocks sooner or later. 
The Filcoin block time is 30 seconds per block and the loop fetches up to all blocks between the current database and the head (minus finality).